dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/CWLTestParameterFileHelper.java

```diff
@@ -95,9 +96,17 @@ public List<FileInput> fileInputs(JSONObject parameterFile) {
     }
 
     private List<String> getPaths(JSONObject fileJsonObject) {
-        final String path = fileJsonObject.getString(PATH);
-        final List<String> secondaryFiles = getSecondaryFiles(fileJsonObject);
-        return Stream.concat(Stream.of(path), secondaryFiles.stream()).toList();
+        if (fileJsonObject.has(PATH) || fileJsonObject.has(LOCATION)) {
```

Comment:
This code works and doesn't need to be changed, but worth noting that `JSONObject` has a set of `opt` functions that you can use to include fallback values:
https://developer.android.com/reference/org/json/JSONObject#optString(java.lang.String,%20java.lang.String)
With those, you could do something like:
```
String path = fileJsonObject.optString(PATH, fileJsonObject.optString(LOCATION, null));
if (path != null) {
  [...]
}
```
Not sure if it's clearer but definitely shorter and probably harder to break.

Reply:
Nice